### PR TITLE
Update to SUIR 2.0.0 and add facade for LabelGroup

### PR DESCRIPTION
**Pull request by cquiroz**
Signed-off-by: Carlos Quiroz <carlos.m.quiroz@gmail.com>

### CODE REVIEW

File: facade/src/main/scala/react/semanticui/modules/popup/Popup.scala [hunk 1]
```diff
@@ -249,7 +253,7 @@ object Popup {
     hideOnScroll.foreach(v => p.hideOnScroll = v)
     hoverable.foreach(v => p.hoverable = v)
     inverted.foreach(v => p.inverted = v)
-    offset.foreach(v => p.offset = v)
+    offset.foreach(v => p.offset = js.Array(v._1, v._2))
```
Comment (rpiaggio) [inline facade/src/main/scala/react/semanticui/modules/popup/Popup.scala:256]: Can we use the implicit conversion from `js.Tuple2` here?

File: facade/src/main/scala/react/semanticui/modules/popup/Popup.scala [hunk 2]
```diff
@@ -27,7 +27,7 @@ final case class Popup(
   hideOnScroll:           js.UndefOr[Boolean] = js.undefined,
   hoverable:              js.UndefOr[Boolean] = js.undefined,
   inverted:               js.UndefOr[Boolean] = js.undefined,
-  offset:                 js.UndefOr[JsNumber | String] = js.undefined,
+  offset:                 js.UndefOr[(JsNumber, JsNumber)] = js.undefined,
```
Comment (rpiaggio) [inline facade/src/main/scala/react/semanticui/modules/popup/Popup.scala:30]: Did this change in a new version of `Popper`?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: facade/src/main/scala/react/semanticui/modules/popup/Popup.scala (hunk 2 above)
Comment (cquiroz) [reply facade/src/main/scala/react/semanticui/modules/popup/Popup.scala:30]: Yes it now takes a tuple or a custom function but I didn't facade that

**Pull request merged**